### ISSUE TITLE
Tsickle distribution package contains multiple amd-module declaratios. 

### ISSUE THREAD

**Issue by mhevery** [angular/tsickle#846]
The v0.30.0 distribution of Tsickle is invalid. some files contain `amd-module` declarations more than once. Having multiple declarations of `amd-module` is invalid TypeScript file.

Example: `node_modules/tsickle/src/googmodule.d.ts`
```
/// <amd-module name="tsickle/src/googmodule" />
/**
 * @license
 * Copyright Google Inc. All Rights Reserved.
 *
 * Use of this source code is governed by an MIT-style license that can be
 * found in the LICENSE file at https://angular.io/license
 */
/// <amd-module name="tsickle/src/googmodule" />
import {ModulesManifest} from './modules_manifest';
import * as ts from './typescript';

```

Here is a full list of offending files:

```
node_modules/tsickle/src/googmodule.d.ts:9:1 - error TS2458: An AMD module cannot have multiple name assignments.

9 /// <amd-module name="tsickle/src/googmodule" />
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


node_modules/tsickle/src/modules_manifest.d.ts:9:1 - error TS2458: An AMD module cannot have multiple name assignments.

9 /// <amd-module name="tsickle/src/modules_manifest" />
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


node_modules/tsickle/src/source_map_utils.d.ts:9:1 - error TS2458: An AMD module cannot have multiple name assignments.

9 /// <amd-module name="tsickle/src/source_map_utils" />
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


node_modules/tsickle/src/tsickle.d.ts:9:1 - error TS2458: An AMD module cannot have multiple name assignments.

9 /// <amd-module name="tsickle/src/tsickle" />
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


node_modules/tsickle/src/typescript.d.ts:9:1 - error TS2458: An AMD module cannot have multiple name assignments.

9 /// <amd-module name="tsickle/src/typescript" />
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```

**Comment by evmar** [comment]
Amusingly, we don't write this comment, so it might be a bug in how TypeScript generates its d.ts files (?)

**Comment by rkirov** [comment]
It's a bug in the bazel ts_library build rules - somewhere around this https://github.com/bazelbuild/rules_typescript/blob/c7b6880767eaae44d8777a7a1b137de92b8fd4d9/internal/tsc_wrapped/compiler_host.ts#L385

**Comment by evmar** [comment]
Nice find!  It looks like the bug it was working around was fixed in TS 2.9, so that's why it double-emits.
